packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -210,6 +212,50 @@ class _CodeViewState extends State<CodeView> with AutoDisposeMixin {
   Widget buildCodeArea(BuildContext context) {
     final theme = Theme.of(context);
 
+    final lines = <TextSpan>[];
+    final style = theme.textTheme.bodyText2.copyWith(
+      fontFamily: 'RobotoMono',
+    );
```

Comment:
use `fixedFontStyle` from theme.dart instead. Feel free to add an optional param for fontSize that defaults to 13.0 if you need a different size.